src/auth.js

```diff
@@ -21,11 +22,14 @@ router.post ('/login', middleware.isNotAuthenticated, (req, res, next) => {
     return;
   }
 
-  models.User.find({where : {
-    email, password
-  }}).then(user => {
-    if (user) sendUserData(user, res);
-    else res.sendStatus(401);
+  models.User.find({where : { email }}).then(user => {
+    if (user) {
+      bcrypt.compare(password, user.password, function(err, response) {
+        if (err) return next (err);
+        if (response == true) sendUserData(user, res);
+        else res.sendStatus(401);
+      });
+    }
```

Comment:
Alsso `else res.sendStatus (401)` here meaning bad crendentials since wrong email. but we donot provide a way to know this in frontend (wrong email or wrong password both shoud give 401)